CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `DataArray.to_hdf5()` accepts both file handles and file paths.
 
 ### Fixed
+- Add dispersion information to dataframe output when available from moed solver.
```

Comment:
moed -> mode. Maybe also mention "dispersion (ps/(nm km))" in here so users know where to look?